gui/src/lib/color.ts

```diff
@@ -13,13 +13,21 @@ export const textHoverColor = (deg: number) => `hsl(${deg}, 95%, 20%)`;
 export const bgColor = (deg: number) => `hsl(${deg}, 65%, 92%)`;
 export const bgHoverColor = (deg: number) => `hsl(${deg}, 65%, 87%)`;
 
+export const darkTextColor = (deg: number) => `hsl(${deg}, 75%, 75%)`;
+export const darkTextHoverColor = (deg: number) => `hsl(${deg}, 75%, 90%)`;
+export const darkBgColor = (deg: number) => `hsl(${deg}, 80%, 6%)`;
+export const darkBgHoverColor = (deg: number) => `hsl(${deg}, 80%, 12%)`;
+
 // This computes a HTML style attribute string for colored logs.
 export const logStyleFromHash = (s: string) => {
```

Comment:
Ideally, this would take some sort of `theme: Theme` argument, so that all the color handling logic for a theme can be centralized, rather than spread out all over the place. Not pressing now though.

Reply:
This kinda causes some issues with spreading out how themes are handled @brandonbloom 

Although, I think it may be necessary anyways. Looking at the Monaco editor, it looks like the theme needs to be set by the JS code and cannot be set simply by a CSS media query :/

Reply:
Never mind I got the Monaco editor working